commands/incr.md

```diff
@@ -154,6 +154,8 @@ ELSE
 END
 ```
 
+**The above code is not safe for concurrent usage**
```

Comment:
this problem can be fixed too, just add length checking after calling the RPUSH and before PERFORM_API_CALL()